pdsfile/pds4file/rules/uranus_occs_earthbased.py

```diff
@@ -544,235 +467,57 @@ def FILENAME_KEYLEN(self):
-        'volumes/COISS_2xxx/COISS_2099/data/1820648744_1820668365/N1820648744_1.IMG',
-        'volumes/COISS_2xxx/COISS_2100/data/1822395235_1823032096/N1822395235_1.IMG',
-        'volumes/COISS_2xxx/COISS_2101/data/1829539156_1829548178/N1829539156_1.IMG',
-        'volumes/COISS_2xxx/COISS_2101/data/1829796937_1830040346/N1830032820_1.IMG',
-        'volumes/COISS_2xxx/COISS_2102/data/1830403828_1831023635/N1830403828_1.IMG',
-        'volumes/COISS_2xxx/COISS_2103/data/1837313417_1837374306/N1837313417_1.IMG',
-        'volumes/COISS_2xxx/COISS_2104/data/1838169062_1838275771/N1838169062_1.IMG',
-        'volumes/COISS_2xxx/COISS_2104/data/1839950576_1840263036/N1840141265_1.IMG',
-        'volumes/COISS_2xxx/COISS_2105/data/1844319700_1844455199/N1844319700_1.IMG',
-        'volumes/COISS_2xxx/COISS_2106/data/1846032655_1846125258/N1846032655_1.IMG',
-        'volumes/COISS_2xxx/COISS_2106/data/1849924886_1850088089/W1850083457_1.IMG',
-        'volumes/COISS_2xxx/COISS_2107/data/1851331524_1851342631/N1851331524_1.IMG',
-        'volumes/COISS_2xxx/COISS_2108/data/1854009318_1854287039/N1854009318_1.IMG',
-        'volumes/COISS_2xxx/COISS_2109/data/1859795917_1860623101/N1859795917_1.IMG',
-        'volumes/COISS_2xxx/COISS_2109/data/1859795917_1860623101/N1860474403_7.IMG',
-        'volumes/COISS_2xxx/COISS_2110/data/1861975118_1862056749/N1861975118_1.IMG',
-        'volumes/COISS_2xxx/COISS_2111/data/1866071296_1866225122/N1866071296_1.IMG',
-        'volumes/COISS_2xxx/COISS_2112/data/1869784516_1869980408/N1869784516_1.IMG',
-        'volumes/COISS_2xxx/COISS_2112/data/1869980499_1870085550/N1870058057_1.IMG',
-        'volumes/COISS_2xxx/COISS_2113/data/1873427694_1873496724/N1873427694_1.IMG',
-        'volumes/COISS_2xxx/COISS_2114/data/1876494184_1876764383/W1876494184_3.IMG',
-        'volumes/COISS_2xxx/COISS_2115/data/1877634459_1877840382/N1877634459_1.IMG',
-        'volumes/COISS_2xxx/COISS_2115/data/1879964192_1880041598/N1880041282_1.IMG',
-        'volumes/COISS_2xxx/COISS_2116/data/1881830414_1881948814/N1881830414_1.IMG',
-    ]
+ 	'bundles/uranus_occs_earthbased/uranus_occ_u0_kao_91cm/data/atmosphere/u0_kao_91cm_734nm_counts-v-time_atmos_ingress.tab',
+        'bundles/uranus_occs_earthbased/uranus_occ_u0_kao_91cm/data/atmosphere/u0_kao_91cm_734nm_counts-v-time_atmos_egress.tab',
+        'bundles/uranus_occs_earthbased/uranus_occ_u0_kao_91cm/data/global/u0_kao_91cm_734nm_counts-v-time_occult.tab',
+ 	'bundles/uranus_occs_earthbased/uranus_occ_u0_kao_91cm/data/global/u0_kao_91cm_734nm_radius_equator_egress_100m.tab', # highest-resolution used as primary
+  	'bundles/uranus_occs_earthbased/uranus_occ_u0_kao_91cm/data/global/u0_kao_91cm_734nm_radius_equator_ingress_100m.tab',
+   	'bundles/uranus_occs_earthbased/uranus_occ_u0_kao_91cm/data/rings/u0_kao_91cm_734nm_counts-v-time_rings_egress.tab',
+   	'bundles/uranus_occs_earthbased/uranus_occ_u0_kao_91cm/data/rings/u0_kao_91cm_734nm_counts-v-time_rings_ingress.tab',
```

Comment:
we don't need to worry about the opus id to primary filespec conversion for `.*counts-v-time_occult.*` under `/rings` because they won't be in the index file